grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/utils/CallerUtils.java

```diff
@@ -82,6 +82,8 @@ public static Class getCallerClass(String key, String value) {
             String valueCap;
             if (value.equals("byte[]")) {
                 valueCap = "Bytes";
+            } else if (value.equals("time:Utc")) {
+                valueCap = "Timestamp";
```

Comment:
Shall we make it camel case (`TimeStamp`)?

Reply:
isnt the standard word `Timestamp`?

Reply:
Yeah, this is fine.